Makefile

```diff
@@ -133,6 +134,19 @@ test-sim-backwards-compatible: ## Test uploading a file with lastest release (je
 
 ##@ Build
 
+.PHONY: storagenode-web
+storagenode-web:
+	# install npm dependencies
+	cd web/storagenode; npm ci
```

Comment:
I'd prefer this was built in a container locked to a specific npm version like the way the binaries are built and the way the console is currently built.

Reply:
okay so this should have a `docker run --rm -i -v "${PWD}":/go/src/storj.io/storj -w /go/src/storj.io/storj` type prefix like when we call release.sh later on. does storjlabs/golang have npm installed in it?

Reply:
perhaps we don't need to use our own image and we can use the official NodeJS docker image

Reply:
No, `storjlabs/golang` doesn't have npm installed in it. We should use `node:10.15.1` like we have been.